<launch>
  <node name="webcam" pkg="usb_cam" type="usb_cam_node" >
    <param name="pixel_format" value="yuyv" />
  </node>

  <node name="maVisu" pkg="image_view" type="image_view" output="screen" >
    <remap from="image" to="/webcam/image_raw" />
  </node>

  <node name="cam_sift" pkg="tp_ros_ensta_u2is" type="cam_sift_cpp_node">
    <remap from="image_topic_in" to="/webcam/image_raw" />
  </node>

  <node name="maVisuOut" pkg="image_view" type="image_view" output="screen" >
    <remap from="image" to="/image_topic_out" />
  </node>

</launch>
